src/main/kotlin/io/github/inductiveautomation/kindling/log/models.kt

```diff
@@ -71,52 +95,69 @@ class SystemLogsColumns(panel: LogPanel) : ColumnList<SystemLogsEvent>() {
             )
             comparator = compareBy(AlphanumComparator(), valueExtractor)
         },
-        value = SystemLogsEvent::logger,
+        getValue = LogEvent::logger,
+    )
+
+    val Message = Column<T, String>(
+        header = "Message",
+        getValue = LogEvent::message,
     )
 
-    val Message by column { it.message }
+    init {
+        add(Marked)
+        add(Level)
+        add(Logger)
+        add(Message)
+        add(Timestamp)
+    }
+
+    abstract val filterableColumns: List<Column<T, out Any?>>
+    abstract val markableColumns: List<Column<T, out Any?>>
 }
 
-@Suppress("unused", "PropertyName")
-class WrapperLogColumns(panel: LogPanel) : ColumnList<WrapperLogEvent>() {
-    val Level by column(
-        column = {
-            minWidth = 55
-            maxWidth = 55
+@Suppress("PropertyName")
+class SystemLogColumns(panel: LogPanel) : LogColumnList<SystemLogEvent>(panel) {
+    val Thread = Column(
+        header = "Thread",
+        columnCustomization = {
+            minWidth = 50
         },
-        value = { it.level },
+        getValue = SystemLogEvent::thread,
     )
-    val Timestamp by column(
-        column = {
-            minWidth = 155
-            maxWidth = 155
-            cellRenderer = DefaultTableRenderer {
-                panel.dateFormatter.format(it as Instant)
-            }
-        },
-        value = { it.timestamp },
+
+    init {
+        add(Thread)
+    }
+
+    override val filterableColumns = listOf(
```

Comment:
Both subclasses have the exact same values in these lists - can be combined into one?